charts/stellar-disbursement-platform/templates/05.2-secrets-ap.yaml

```diff
@@ -13,10 +13,12 @@ metadata:
     {{- toYaml .Values.anchorPlatform.kubeSecrets.annotations | nindent 4 }}
   {{- end }}
 
-{{- if .Values.anchorPlatform.configMap.data }}
+{{- if .Values.anchorPlatform.kubeSecrets.data }}
```

Comment:
This was the error with Anchor where we were populating the secrets with Configmap values. Didn't cause issues for our deploys because we created secrets externally from Vault.